### ISSUE TITLE
Jekyll support

### ISSUE THREAD

**Issue by StefanoCecere** [shd101wyy/markdown-preview-enhanced#308]
in the CHANGELOG you write:
- add blog (jekyll, hexo, etc...) support. 

what does it mean?
(i use Jekyll and would be interested in helping this feature)

**Comment by shd101wyy** [comment]
Hi @StefanoCecere ,  

Yes I am planning to implement support for blog engines by adding commands `create new post`.   
But I don't know where to start. Do you have any advice.   

Thanks

**Comment by StefanoCecere** [comment]
well.. yes i'm using markdown-writer and it works quite well (also its toolbar https://github.com/zhuochun/tool-bar-md-writer !

so, maybe better to integrate well with it, suggesting it and not replicating its features inside markdown-preview-enhanced

maybe you can integrate its features in the future, the day that it worn't be supported anymore!

(in its toolbar it tries to open the "normal md previewer" and it doesnt work with markdown-preview-enhanced (i made a custom button for us)

maybe i could ask its developer to support markdown-preview-enhanced

**Comment by shd101wyy** [comment]
Ok i see.

**Comment by StefanoCecere** [comment]
well.. i see the two plugins start to have some features in common (insert table/images for example)
markdown-preview-enhanced is entering the "editor" side...
i guess you should decide if to leave the editor features to markdown-writer, or if to become a full edit&preview suite!

did you already talk with https://atom.io/users/zhuochun ?

**Comment by shd101wyy** [comment]
No, I think I will stop this plan for now and try his package first.
